client/package.json

```diff
@@ -14,7 +14,8 @@
     "build:dev:server": "webpack -w --config webpack.server.rails.build.config.js",
     "hot-assets": "babel-node server-rails-hot.js",
     "print-phantomjs-version": "phantomjs --version",
-    "start-phantomjs": "phantomjs --webdriver=8910"
+    "start-phantomjs": "phantomjs --webdriver=8910",
+    "storybook": "start-storybook -p 9001"
```

Comment:
Could this script name be `styleguide` instead of `storybook`? Might be easier to remember since we talk about Styleguide anyways, and would also allow to change the tech under the hood without touching the script name (although not a big problem really).